deployment_and_ssl_doc.md

```diff
@@ -39,14 +55,24 @@ apt-get install elixir
 
 #### Install Hex
 ```
-mix local hex
+mix local.hex
 ```
 
 #### Install Phoenix
 ```
 mix archive.install https://github.com/phoenixframework/archives/raw/master/phoenix_new.ez --force
 ```
 
+#### Install tools to build release with edeliver
+`wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && sudo dpkg -i erlang-solutions_1.0_all.deb`
```

Comment:
Good addition in all as I could see edeliver was not building the release of the app, however, isn't this command already run when installing elixir? Is there any reason that it needs to be run again?

Reply:
As per @RobStallion's comment, isn't this already installed in the section **Install Elixir**?

Reply:
Yep, I can remove this.